samba/data/smb.conf

```diff
@@ -0,0 +1,19 @@
+[global]
+        security = user
+        server min protocol = SMB2
+
+        # disable printing services
+        load printers = no
+        printing = bsd
+        printcap name = /dev/null
+        disable spoolss = yes
+
+[Data]
+        path = /storage
+        comment = Shared
+        valid users = @smb
+        browseable = yes
+        writable = yes
+        read only = no
+        force user = root
+        force group = root
```

Comment:
I am not an expert in samba, but with the reduced permissions to user 1000 and group 1000 we probably need to change it here too (or remove it completly)